test/evergreen.yml

```diff
@@ -5350,12 +5394,10 @@ buildvariants:
     - name: perf-test-medium-btree-backup
     - name: perf-test-mongodb-oplog
     - name: perf-test-mongodb-small-oplog
-    # FIXME-BUILD-16428: Re-enable once snappy is installed on mac platforms.
```

Comment:
Let's leave this commented out. There's a good chance when BUILD-164268 is resolved we'll forget to add `secondary-apply` back to this list the first time we run a patch.

Reply:
Good point, thanks! [6c349a4](https://github.com/wiredtiger/wiredtiger/pull/8525/commits/6c349a4fd0ec741744132000d8acf38c39733320)